src/main/kotlin/nl/info/zac/formio/FormioService.kt

```diff
@@ -1,17 +1,16 @@
 /*
- * SPDX-FileCopyrightText: 2024 Dimpact
+ * SPDX-FileCopyrightText: 2025 Lifely
```

Comment:
```suggestion
 * SPDX-FileCopyrightText: 2025 INFO
```

Reply:
No, we still use Lifely in our SPDX copyright headers. Also see CONTRIBUTING.md. If we wish to change this (I'm not sure we should) we should first discuss this and then change them all.

Reply:
But this change is not ok because I accidentally removed Dimpact. Will fix!